extensions/gdx-freetype/src/com/badlogic/gdx/graphics/g2d/freetype/FreeTypeFontGenerator.java

```diff
@@ -161,11 +161,15 @@ public BitmapFont generateFont (FreeTypeFontParameter parameter, FreeTypeBitmapF
 		if (updateTextureRegions)
 			parameter.packer.updateTextureRegions(data.regions, parameter.minFilter, parameter.magFilter, parameter.genMipMaps);
 		if (data.regions.isEmpty()) throw new GdxRuntimeException("Unable to create a font with no texture regions.");
-		BitmapFont font = new BitmapFont(data, data.regions, true);
+		BitmapFont font = newBitmapFont(data, data.regions, true);
 		font.setOwnsTexture(parameter.packer == null);
 		return font;
 	}
-
+	
+	protected BitmapFont newBitmapFont(BitmapFont.BitmapFontData data, Array<TextureRegion> pageRegions, boolean integer) {
+		return new BitmapFont(data, pageRegions, true);

```

Comment:
it should use integer parameter instead of true i think

Reply:
Also, how about a better name or at least javadoc on that `integer` parameter.

Reply:
Also, how about a better name or at least javadoc on that `integer` parameter.